brave/brave-core
# Adding option to restore excluded pubs

Fixes: https://github.com/brave/brave-browser/issues/936

# Submitter Checklist:


## Test Plan:
1. Build Brave from this branch, and start with a clean profile
2. Enable rewards
3. Visit 7 - 10 sites so that they get added to the contribution table
4. Exclude some sites.
5. Hit the `Restore` action under the modal title
6. Ensure the excluded sites are restored.
7. Ensure that those sites can be re-excluded then restored.

## Reviewer Checklist:
